cli/command/volume/create.go

```diff
@@ -37,6 +55,7 @@ func newCreateCommand(dockerCli command.Cli) *cobra.Command {
 				}
 				options.name = args[0]
 			}
+			options.cluster = hasClusterVolumeOptionSet(cmd.Flags())
```

Comment:
I think creating a cluster volume should be explicitly set by the user, rather than implied by setting any of the cluster-volume options so that users don't unintentionally create cluster volumes by setting a generically-named flag. Strawman: the `--group` flag pulls double-duty as the "create a cluster volume" flag by renaming it to <tt>&#8209;&#8209;cluster&#8209;volume&nbsp;_GROUPNAME_</tt> and making it an error to set any of the other cluster-only options unless that flag is also set. If a user wanted for whatever reason to create a cluster volume in the empty-string group, they could do so explicitly: <tt>docker volume create &#8209;&#8209;cluster&#8209;volume&nbsp;""</tt>